kura/org.eclipse.kura.core.comm/src/main/java/org/eclipse/kura/core/comm/CommConnectionImpl.java

```diff
@@ -44,7 +46,7 @@
             String existingDirs = System.getProperty("java.ext.dirs");
```

Comment:
Would be good if you declare `java.ext.dirs` as a constant so that it can be reused in different places without duplicating.